### DISCUSSION TITLE
Python logging field disabled

### DISCUSSION THREAD

**Discussion by Jedore** [apache/skywalking#10360] [General]
In `logging.Logger.handle` method,  execute a judge about `disabled`. 
```python
    def handle(self, record):
        """
        Call the handlers for the specified record.

        This method is used for unpickled records received from a socket, as
        well as those created locally. Logger-level filtering is applied.
        """
        if (not self.disabled) and self.filter(record):
            self.callHandlers(record)
```
Maybe there will be a situation in the future: `self.fitler(record) == True` and `self.disabled == True`. If this, then won't execute `self.callHandlers(record)`, but `sw_logging._sw_handle` won't filter this record by `self.disabled`. 

Do we need to optimize this logic?

**Comment by Superskyyy** [comment]
@Jedore You mean we didn't check if `self.disabled` so the correct behavior is no matter if the record was filtered or not it should always directly return when handler is disabled right? I think this can be added, I overlooked this one.

**Comment by Jedore** [comment]
> @Jedore You mean we didn't check if `self.disabled` so the correct behavior is no matter if the record was filtered or not it should always directly return when handler is disabled right? I think this can be added, I overlooked this one.

Sure. I do think so.

**Reply by Superskyyy** [reply]
Would you like to help contribute this? I'm working on finishing my feature branch for the release.

**Comment by Jedore** [comment]
Ok, just a little change, i would make it.@Superskyyy
